boilerplate/fullstack/next-app-dir/app/components/home.tsx

```diff
@@ -21,14 +21,14 @@ export async function HomePage() {
         }
 
         if (hasInvalidClaims) {
-            return <SessionAuthForNextJS />;
+            return <SessionAuthForNextJS ssrSessionExists={false} />;
```

Comment:
technically, the session exists here, it's just that the claims are invalid. So the prop's name `ssrSessionExists` doesn't fit.